bin/update-dep-versions.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// update all the dependencies inside packages/node_modules/*/package.json to reflect
+// the version numbers in the top-level package.json
+
+var fs = require('fs');
+
+var topPkg = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+
+var modules = fs.readdirSync('./packages/node_modules');
+modules.forEach(function (mod) {
+  var pkgPath = './packages/node_modules/' + mod + '/package.json';
+  var pkg = JSON.parse(fs.readFileSync(pkgPath, 'utf8'));
+  var depLists = [
+    pkg.dependencies, 
+    pkg.peerDependencies, 
+    pkg.optionalDependencies
+  ];
+  depLists.forEach(function (deps) {
+    if (!deps) {
+      return;
+    }
+    Object.keys(deps).forEach(function (dep) {
+      if (topPkg.dependencies[dep]) {
+        deps[dep] = topPkg.dependencies[dep];
+      } else { // core pouchdb-* module
+        deps[dep] = topPkg.version;
```

Comment:
I'd check for startsWith `"pouchdb"` and throw an error otherwise, to prevent forgotten dependencies being defined with a random version.